tools/write_release.py

```diff
@@ -0,0 +1,121 @@
+"""
+Standalone script for writing release doc::
+
+    python tools/write_release <version>
+
+Example::
+
+    python tools/write_release.py 1.7.0
+
+Needs to be run from the root of the repository and assumes
+that the output is in `release` and wheels and sdist in
+`release/installers`.
+
+Translation from rst to md markdown requires Pandoc, you 
+will need to rely on your distribution to provide that.
+
+"""
+import os
+import subprocess
+import textwrap
+import argparse
+from hashlib import md5
+from hashlib import sha256
+from pathlib import Path
+
+# Name of the notes directory
+NOTES_DIR = "doc/source/release"
+# Name of the output directory
+OUTPUT_DIR = "release"
+# Output base name, `.rst` or `.md` will be appended
+OUTPUT_FILE = "README"
+
+def compute_hash(wheel_dir, hash_func):
+    """
+    Compute hashes of files in wheel_dir.
+
+    Parameters
+    ----------
+    wheel_dir: str
+        Path to wheel directory from repo root.
+    hash_func: function
+        Hash function, i.e., md5, sha256, etc.
+
+    Returns
+    -------
+    list_of_strings: list
+        List of of strings. Each string is the hash
+        followed by the file basename.
+
+    """
+    released = os.listdir(wheel_dir)
+    checksums = []
+    for fn in sorted(released):
+        fn_path = Path(f"{wheel_dir}/{fn}") 
+        with open(fn_path, 'rb') as f:
+            m = hash_func(f.read())
+        checksums.append(f"{m.hexdigest()}  {fn}")
+    return checksums
+
+
+def write_release(version):
+    """
+    Copy the <version>-notes.rst file to the OUTPUT_DIR, append
+    the md5 and sha256 hashes of the wheels and sdist, and produce
+    README.rst and README.md files.
+
+    Parameters
+    ----------
+    version: str
+       Release version, e.g., '2.3.2', etc.
+
+    Returns
+    -------
+    None.
+
+    """
+    notes = Path(f"{NOTES_DIR}/{version}-notes.rst")
+    wheel_dir = Path(f"{OUTPUT_DIR}/installers")
+    target_md = Path(f"{OUTPUT_DIR}/{OUTPUT_FILE}.md")
+    target_rst = Path(f"{OUTPUT_DIR}/{OUTPUT_FILE}.rst")
```

Comment:
nitpick:

```suggestion
    notes = Path(NOTES_DIR) / f"{version}-notes.rst"
    wheel_dir = Path(OUTPUT_DIR) / "installers"
    target_md = Path(OUTPUT_DIR) / f"{OUTPUT_FILE}.md")
    target_rst = Path(OUTPUT_DIR) / "{OUTPUT_FILE}.rst"
```

but I suppose this could be cleaned up a bit by defining `NOTES_DIR` etc. directly as a `Path`